connectors/soundcloud.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+import BaseConnector from 'content/base-connector';
+import Utils from 'content/utils';
+
+new class extends BaseConnector {
+    constructor() {
+        super();
+        this.name = 'SoundCloud';
+        this.prefix = '/com/soundcloud';
+
+        this.artistSelector = '.playbackSoundBadge .playbackSoundBadge__lightLink';
+        this.titleSelector = '.playbackSoundBadge .playbackSoundBadge__titleLink span[aria-hidden=true]';
+
+        this.currentTimeSelector = '.playbackTimeline__timePassed span[aria-hidden=true]';
+        this.lengthSelector = '.playbackTimeline__duration span[aria-hidden=true]';
+        this.playButtonSelector = '.playControls__elements .playControls__play';
+        this.prevButtonSelector = '.playControls__elements .skipControl__previous';
+        this.nextButtonSelector = '.playControls__elements .skipControl__next';
+        this.artSelector = '.playbackSoundBadge__avatar span.sc-artwork';
+
+        Utils.query('.playControls .playControls__inner').then(elem => this.observe(elem));
+    }
+
+    get playbackStatus() {
+        return Utils.query(this.playButtonSelector).then(elem => elem.classList.contains('playing') ? 'playing' : 'paused');
+    }
+
+    transform_track_to_artist() {

```

Comment:
First of all, I use camelCase for functions and variables in the whole project, so I think it's better to follow the same code style across all of it.
Secondly, I think `transfrom_track_to_artist` isn't a good name for a method that returns `{artist, track}`. I'd consider something like `getArtistTrack()`.